web/viewer.css

```diff
@@ -334,8 +334,7 @@ body {
 }
 
 :is(.toolbar, .editorParamsToolbar, #sidebarContainer)
-  :is(input, button, select),
-.secondaryToolbar :is(input, button, a, select) {
```

Comment:
Note that the secondary toolbar doesn't have any `input` or `select` elements inside it, so those two have been removed in the new rule.